distributionviewer/core/static/js/app/components/views/example-redux-chart.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+
+export default function(props) {
+  return (
+    <pre>
+      <code>
+        {props.charts.map(chart => {
+          return (<p>{chart.name}</p>);
+        })}
+      </code>
+    </pre>
```

Comment:
I notice we're using &lt;pre&gt; and &lt;code&gt; here. Is that for something down the road?